### PR TITLE
fix(VMenu): fix min width

### LINKED ISSUE #16987: [Bug Report][3.1.11] v-menu min-width property does not work
### Environment
**Vuetify Version:** 3.1.11
**Vue Version:** 3.2.47
**Browsers:** Chrome 111.0.0.0
**OS:** Linux x86_64

### Steps to reproduce
Click on the button to open the menu

### Expected Behavior
The menu should have 500px width because it has min-width="500"

### Actual Behavior
min-width property is not respected

### Reproduction Link
[https://codepen.io/kasheftin/pen/dyqQgJV?editors=1010](https://codepen.io/kasheftin/pen/dyqQgJV?editors=1010)

### PR DESCRIPTION
**Pull request by lzl0304**
## Description

fixes #16987 

## Markup:

```vue
<template>
  <v-container>
    <div class="text-center">
      <v-menu>
        <template v-slot:activator="{ props }">
          <v-btn v-bind="props">
            Auto width
          </v-btn>
        </template>
        <v-list>
          <v-list-item>
            <v-list-item-title>Here</v-list-item-title>
          </v-list-item>
        </v-list>
      </v-menu>
      <v-menu min-width="500">
        <template v-slot:activator="{ props }">
          <v-btn v-bind="props">
            Min width
          </v-btn>
        </template>
        <v-list>
          <v-list-item>
            <v-list-item-title>Here</v-list-item-title>
          </v-list-item>
        </v-list>
      </v-menu>
    </div>
  </v-container>
</template>
```

### CODE REVIEW

**Comment by websitevirtuoso** [comment]
looks good

File: packages/vuetify/src/components/VOverlay/VOverlay.tsx [hunk 1]
```diff
@@ -254,6 +256,21 @@ export const VOverlay = genericComponent<OverlaySlots>()({
       emit('afterLeave')
     }
 
+    const mergedStyle = computed(() => {
```
Comment (johnleider) [inline packages/vuetify/src/components/VOverlay/VOverlay.tsx:259]: Wouldn't it be easier to just change the style ordering?

image

Comment (lzl0304) [reply packages/vuetify/src/components/VOverlay/VOverlay.tsx:259]: If only the order is changed, when `min-width` is not set in `v-menu`, `min-width: undefined` in `dimensionStyles` will override the default `min-width` in `contentStyles`, so I fixed it this way.

Comment (johnleider) [reply packages/vuetify/src/components/VOverlay/VOverlay.tsx:259]: Your proposed fix just seems to be overly complex on the surface. Surely there is a less engineered way to approach this. I'll think on it.

**Comment by johnleider** [comment]
This Pull Request is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Pull request closed without merging**